Client/Frontend/Strings.swift

```diff
@@ -9,7 +9,6 @@ import Foundation
  Strings with the following IDs may be removed after confirming with Product:
  - `ActivityStream.Pocket.SectionTitle`      --> replaced by `FirefoxHome.Pocket.SectionTitle`
  - `ActivityStream.Pocket.SectionTitle2`    --> replaced by `FirefoxHome.Pocket.SectionTitle`
- - All Strings marked under "unused" in AppMenu struct
  - "HistoryPanel.ClearHistoryMenuTitle" string
```

Comment:
This seems still in use?

Definition: https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Strings.swift#L1691-L1695
Use: https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Library/ClearHistorySheetProvider.swift#L34

Reply:
Gosh, this is so confusing.
* There is a `LibraryPanel.History.ClearHistoryMenuTitle` (string ID: `LibraryPanel.History.ClearHistoryMenuTitle.v100`). 
* And there's a `.ClearHistoryMenuTitle` (string ID: `HistoryPanel.ClearHistoryMenuTitle`), which is unused.